src/validators/__tests__/validateFinalGrade.test.ts

```diff
@@ -0,0 +1,32 @@
+import { describe, it, expect } from 'vitest'
+
+import { validateFinalGrade } from '@/validators/studentsValidators'
+
+describe('validateFinalGrade', () => {
+  it('should return true for valid grades', () => {
+    expect(validateFinalGrade('1')).toBe(true)
+    expect(validateFinalGrade('2')).toBe(true)
+    expect(validateFinalGrade('3')).toBe(true)
+    expect(validateFinalGrade('4')).toBe(true)
+    expect(validateFinalGrade('5')).toBe(true)
```

Comment:
imo too many, anyway if you want to test like this, you can use a loop instead of copying